### PR TITLE
feat(lb): add new data_source lb pools

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

add new data_source lb pools

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add new data_source lb pools
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/lb' TESTARGS='-run TestAccDataLBPools_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lb -v -run TestAccDataLBPools_basic -timeout 360m -parallel 4
=== RUN   TestAccDataLBPools_basic
=== PAUSE TestAccDataLBPools_basic
=== CONT  TestAccDataLBPools_basic
--- PASS: TestAccDataLBPools_basic (100.72s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lb        100.814s
```

### CODE REVIEW

File: docs/data-sources/lb_pools.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+* `admin_state_up` - The administrative state of the pool. A valid value is true (UP) or false (DOWN).
+
+* `healthmonitor_id` - The health monitor ID of the LB pool.
+
+* `listeners` - The listener list. The [object](#elem_object) structure is documented below.
+
+* `loadbalancers` - The loadbalancer list. The [object](#elem_object) structure is documented below.
+
+* `members` - The member list. The [object](#elem_object) structure is documented below.
```
Comment (ShiChangkuo) [inline docs/data-sources/lb_pools.md:70]: using the same tags

File: docs/data-sources/lb_pools.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+* `name` - The pool name.
+
+* `description` - The description of pool.
+
+* `protocol` - The protocol of pool.
+
+* `lb_method` - The load balancing algorithm to distribute traffic to the pool's members.
+
+* `admin_state_up` - The administrative state of the pool. A valid value is true (UP) or false (DOWN).
```
Comment (ShiChangkuo) [inline docs/data-sources/lb_pools.md]: this parameter should be removed.

Comment (Jason-Zhang9309) [reply docs/data-sources/lb_pools.md]: fixed

**Comment by Jason-Zhang9309** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/lb' TESTARGS='-run TestAccDataLBPools_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lb -v -run TestAccDataLBPools_basic -timeout 360m -parallel 4
=== RUN   TestAccDataLBPools_basic
=== PAUSE TestAccDataLBPools_basic
=== CONT  TestAccDataLBPools_basic
--- PASS: TestAccDataLBPools_basic (102.28s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lb        102.425s
```

**Pull request merged**